tests/include/detray/test/cpu/detector_scan.hpp

```diff
@@ -170,15 +170,24 @@ class detector_scan : public test::fixture_base<> {
         const std::size_t n_helices{trk_state_generator.size()};
         intersection_traces.reserve(n_helices);
 
-        const auto pT_range = m_cfg.track_generator().mom_range();
-        std::string mometum_str{std::to_string(pT_range[0]) + "_" +
-                                std::to_string(pT_range[1])};
+        std::string momentum_str{""};
+        if constexpr (!k_use_rays) {
+            const auto pT_range = m_cfg.track_generator().mom_range();
+            // Remove floating point imprecisions
+            momentum_str =
+                std::to_string(
+                    std::floor(10. * static_cast<double>(pT_range[0])) / 10.) +
+                "_" +
+                std::to_string(
+                    std::ceil(10. * static_cast<double>(pT_range[1])) / 10.) +
+                "_GeV";
```

Comment:
Is it intended that this will produce very slightly different files names than before?

Reply:
Yes, the filenames before were not really usable

Reply:
OK!